test/integration/webhook/operatingsystemconfig/resources/doc.go

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//go:generate sh -c "bash $GARDENER_HACK_DIR/generate-crds.sh -p crd- extensions.gardener.cloud resources.gardener.cloud monitoring.coreos.com_v1"
```

Comment:
Why we generate CRDs from the `resources.gardener.cloud` and `monitoring.coreos.com_v1` groups when at the end we are only interested in CRDs from the `extensions.gardener.cloud` group?

```suggestion
//go:generate sh -c "bash $GARDENER_HACK_DIR/generate-crds.sh -p crd- extensions.gardener.cloud"
```